### ISSUE TITLE
Saving package structure

### ISSUE THREAD

**Issue by mirackle-spb** [cmeza20/spring-data-generator#14]
It will be nice if plugin will have feature to save package structure when creating Repos or Managers

**Comment by cmeza20** [comment]
Hi @mirackle-spb 

You mean this ?

![image](https://user-images.githubusercontent.com/9298942/39490821-bf4b53b6-4d4f-11e8-9853-fb4ece43346e.png)

**Comment by mirackle-spb** [comment]
Yep. Exactly :-)

**Comment by mirackle-spb** [comment]
It it possible? :-)

**Comment by cmeza20** [comment]
Hello @mirackle-spb 

Yes, it's.
Will be available in the next release

regards

**Comment by cmeza20** [comment]
Hello @mirackle-spb 

The functionality was published in version 1.1.7    #19 
```
<dependency>
  <groupId>com.cmeza</groupId>
  <artifactId>spring-data-generator</artifactId>
  <version>1.1.7</version>
</dependency>
```
Best regards
Carlos Meza
